source/tyr/tyr/binarisation.py

```diff
@@ -201,15 +211,18 @@ def fusio2ed(self, instance_config, filename, job_id, dataset_uid):
         params.append("--log_comment")
         params.append(instance_config.name)
         res = None
-        with collect_metric('fusio2ed', job, dataset_uid):
+        with collect_metric("fusio2ed", job, dataset_uid):
             res = launch_exec("fusio2ed", params, logger)
         if res != 0:
-            raise ValueError('fusio2ed failed')
+            raise ValueError("fusio2ed failed")
+        dataset.state = "done"
     except:
-        logger.exception('')
-        job.state = 'failed'
-        models.db.session.commit()
+        logger.exception("")
```

Comment:
This will log the call stack with the exception, only with an empty custom message before it. So, yes, it is useful. Maybe there's a better way to do it but it is legacy and done like this everywhere